### DISCUSSION TITLE
Cyberduck Timing Out

### DISCUSSION THREAD

**Discussion by JoWhite76** [iterate-ch/cyberduck#16644] [Q&A]
I have been using Cyberduck for many years without problems but for the last few days I am unable to complete the login in process. After minutes of waiting I get a "Connection Timed Out" messages. With the following options Cancel, Try Again, or Network Diagnostics. Trying again provides the same results. Network Diagnostics does nothing.

I've rang my service provider and they say it's your error.

**Comment by dkocher** [comment]
Please try if changing the following settings resolves the problem:

* Disable firewall in _Settings → Network → Firewall_
* Disable any active network filter in _System Settings → Network → Filters_
* Set _System Settings → Wi-Fi → Details → Private Wi-Fi address → Fixed_
